openml/testing.py

```diff
@@ -24,15 +27,15 @@ class TestBase(unittest.TestCase):
     Hopefully soon allows using a test server, not the production server.
     """
 
-    def setUp(self):
+    def setUp(self, n_levels: int = 1):
```

Comment:
can you please add a tiny comment string explaining `n_levels`? (seems important for understanding unit tests)

Reply:
Yes, sure.